templates/OME-TIFF_Conversion.ipynb

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
<p>Can we add the code to download the example data here (if the data doesn't exist already)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>